extensions/wikia/Recirculation/RecirculationHooks.class.php

```diff
@@ -46,16 +46,21 @@ public static function onBeforePageDisplay( OutputPage $out, Skin $skin ) {
 	public static function onOasisSkinAssetGroups( &$jsAssets ) {
 		$jsAssets[] = 'recirculation_js';
 
+		if ( self::canShowDiscussions() && self::onCorrectPageType() ) {
+			$jsAssets[] = 'recirculation_discussions_js';
+		}
+
 		return true;
 	}
 
 	/**
-	 * Return whether we're on one of the pages where we want to show the Videos Module,
+	 * Return whether we're on one of the pages where we want to show the Recirculation widgets,
 	 * specifically File pages, Article pages, and Main pages
 	 * @return bool
 	 */
-	static public function canShowVideosModule() {
+	static public function onCorrectPageType() {
```

Comment:
The `on` prefixed looks like it's a hook. I would go with `isOnCorrectPapeType` or `isPageTypeCorrect` or something like that.